src/utils.ts

```diff
@@ -7,3 +7,16 @@ export function getNumbers(from: number, to: number): number[] {
 
   return numbers;
 }
+
+export const getVisisibleItems = (
+  itemsPerPage: number,
+  currentPage: number,
+  total: number,
+) => {
+  const lastVisibleItem = currentPage * itemsPerPage;
+
+  return getNumbers(
+    lastVisibleItem - itemsPerPage + 1,
+    (lastVisibleItem > total ? total : lastVisibleItem),
```

Comment:
Now everything is written correctly, but the code is a bit difficult to read and maintain. It is better to use the ternary operator outside the arguments of another function and use it as a ready-made result, for example:
```
const indexFrom = lastVisibleItem - itemsPerPage + 1;
const indexTo= lastVisibleItem > total ? total : lastVisibleItem;
getNumbers(indexFrom, indexTo);
```